examples/tensorflow-mnist-model-inversion/docker-compose.yml

```diff
@@ -213,7 +213,7 @@ services:
     networks:
       - lab_deployment_backend
     volumes:
-      - nfs-datasets:/nfs/data/Mnist
+      - nfs-datasets:/nfs/data
```

Comment:
If the `demo.ipynb` notebook is reverted back (see my comment on `demo.ipynb` file), then this file (`docker-compose.yml`) should reverted back as while.